src/styles/utilities/mixins-styles.less

```diff
@@ -0,0 +1,5 @@
+.candy-stripe(@color-a, @color-b) {
```

Comment:
I'm not sure `mixin-styles.less` is very explanatory. Could we stick to the pattern that we already have and call it `mixins-progressbar.less`?